web/packages/teleport/src/components/AuthnDialog/AuthnDialog.tsx

```diff
@@ -41,13 +41,12 @@ export default function AuthnDialog({
           </Danger>
         )}
         <Text textAlign="center">
-          Re-authentication is required. Follow the prompts given by your
-          browser to complete authentication.
+          Re-enter your multi-factor authentication in the browser to continue.
         </Text>
       </DialogContent>
       <DialogFooter textAlign="center">
-        <ButtonPrimary onClick={onContinue} mr={3} width="130px">
-          {errorText ? 'Retry' : 'Verify'}
+        <ButtonPrimary onClick={onContinue} setRef={e => e && e.focus()} mr={3} width="130px">
```

Comment:
The focus style of the primary button differs from a non-focused button only by the background color, so I suspect that users won't immediately know that the button is already focused.

But I believe changing that is outside of the scope of this PR.

Reply:
Yes, out of scope, but worth mentioning to @roraback for design system updates. I noticed this too.